sapan-parikh/SAPAN_PARIKH.md

```diff
@@ -0,0 +1,16 @@
+# Not All Tests are the Same
+
+I have gotten an opportunity to work on projects where I learned how automated testing is and that not all automated tests are the same. They differ from each other in at least two ways,speed of execution and risk of failure.
+
+While the testing pyramid is QA 101, it'salso one of the most ignored learning. One project that I worked on depended heavily on unit testing, while the other relied heavily on automated() testing. approaches have their shortcomings.
```

Comment:
unit testing is also automated testing though, do you maybe mean end to end testing?